### PR TITLE
(model): add hugging face transformer textattack/albert-base-v2-ag-news example

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
add hugging face transformer example
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: example/transformer/ag_news/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Evaluate Hugging Face Transformer `textattack/albert-base-v2-ag-news` on AG News dataset
+---
+
+```bash
+    git clone https://github.com/star-whale/starwhale.git
```
Comment (tianweidut) [inline example/transformer/ag_news/README.md:6]:
```
    git clone https://github.com/star-whale/starwhale.git
    cd starwhale/client
    python3 -m pip install -e .
```
-->
```
   pip install starwhale
```
use release starwhale pypi package as the user example ?

Comment (anda-ren) [reply example/transformer/ag_news/README.md:6]: > ```
>     git clone https://github.com/star-whale/starwhale.git
>     cd starwhale/client
>     python3 -m pip install -e .
> ```
> 
> -->
> 
> ```
>    pip install starwhale
> ```
> 
> use release starwhale pypi package as the user example ?

This readme directs users to build dataset from code and to build model from code. How could we guarantee the compatibility if user installl a release version of starwhale.

Comment (tianweidut) [reply example/transformer/ag_news/README.md:6]: dev version is not recommended for the users.

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**